engine/flink/util/src/main/scala/pl/touk/nussknacker/engine/flink/util/exception/DefaultEspExceptionHandler.scala

```diff
@@ -62,11 +64,22 @@ case class VerboselyLoggingRestartingExceptionHandler(processMetaData: MetaData,
 case class VerboselyLoggingExceptionConsumer(processMetaData: MetaData, params: Map[String, String] = Map.empty)
   extends FlinkEspExceptionConsumer
     with LazyLogging {
-  override def consume(e: EspExceptionInfo[NonTransientException]) = {
+  override def consume(e: EspExceptionInfo[NonTransientException]): Unit = {
     logger.error(s"${processMetaData.id}: Exception during processing job, params: $params, context: ${e.context}", e.throwable)
   }
 }
 
+case class BrieflyLoggingExceptionConsumer(processMetaData: MetaData, params: Map[String, String] = Map.empty)
+  extends FlinkEspExceptionConsumer
+    with LazyLogging {
+  override def consume(e: EspExceptionInfo[NonTransientException]): Unit = {
+    logger.whenDebugEnabled {
```

Comment:
What do u think about:
```
if (logger.underlying.isDebugEnabled) {
  logger.debug(...)
} else {
  logger.warn(...)
}
```
to avoid double logging? Here and in other places in this change...

Reply:
Hmm... I also thought about it, but when I look at logs I often filter out only WARN/ERROR - and that would get lost if we log only DEBUG here?

Reply:
Good point, it is one of scenario. The second one is that developer just look at logs stream line-by-line. But this grepping scenario looks like more critical. WDYT about extracting some LoggingExt utililty method for that pattern? It will prevent us from incidental refactor of this (by developer who've just perform line-by-line scenario)